presentation-api/index.html

```diff
@@ -97,6 +104,15 @@
   }
 });
 
+document.querySelector('#send').addEventListener('click', () => {
+  if (presentationConnection && presentationConnection.state == 'connected') {
+    const message = document.querySelector('#message').value.trim();
+    const lang = document.body.lang || 'en-US';
+    ChromeSamples.log('Sending ' + message + ' to ' + presentationConnection.id);
+    presentationConnection.send(JSON.stringify({string: message, lang: lang}));
```

Comment:
Noted.
I can see this at https://www.w3.org/TR/presentation-api/#passing-locale-information-with-a-message but I don't get why the remote page needs it. if the appropriate charset is defined, we should be good no? (e.g. ` <meta charset="UTF-8">`)